akka-docs/src/main/paradox/java/scala-compat.md

```diff
@@ -3,10 +3,6 @@
 Akka requires that you have [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html) or
 later installed on your machine.
 
-## Scala and Java APIs
-
-Akka has both a @ref:[Scala Documentation](../scala.md) and a @ref:[Java Documentation](../java.md).
-
 ## Java 8 types
```

Comment:
ah, I see that this page was a java only page, I added some scala stuff here "Do not use -optimize Scala compiler flag". I'll create a ticket for that, I don't know where to place it.